integration/client_test.go

```diff
@@ -263,7 +263,7 @@ func (e *Env) setupHistory(ctx context.Context, userID string, signers []signatu
 	// Note that profile 5 is submitted twice by the user to test that
 	// filtering case.
 	for i, p := range [][]byte{
-		nil, nil, cp(1), cp(2), nil, nil, cp(3), nil,
+		nil, cp(1), cp(2), nil, nil, cp(3), nil,
```

Comment:
The trillian map server starts at revision 1 rather than revision 0. 
This change shifts the test data by one epoch to accomodate that.